currency/currency.go

```diff
@@ -99,19 +96,13 @@ func ToMinorUnit(currencyCode string, amount float64) (minorUnitAmount uint64, e
 // FromMinorUnit converts amount in the smallest unit (minor unit) of a currency to amount in that currency
 func FromMinorUnit(currencyCode string, minorUnitAmount uint64) (amount float64, err error) {
 	if !IsISO4217(currencyCode) {
-		err = fmt.Errorf("%s is not a valid ISO 4217 currency code", currencyCode)
-		return
+		return 0, fmt.Errorf("%s is not a valid ISO 4217 currency code", currencyCode)
 	}
-	switch {
-	case minorUnitAmount > 0:
+
+	if minorUnitAmount > 0 {
```

Comment:
Do we need to worry about minorUnitAmount < 0 ? 😮

Reply:
I think not possible since it the type is uint64

Reply:
Yes, that why I removed it

Reply:
Alright! I will resolve this and +1!